frontend/src/fixedComponents/Slider.js

```diff
@@ -1,15 +1,61 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import logo1 from '../images/1.jpg';
+import logo2 from '../images/2.jpg';
+import logo3 from '../images/3.jpg';
```

Comment:
Instead of keeping the images in a folder, you can get fake images from somewhere like unsplash. at least in development.